src/timer/timer_config.c

```diff
@@ -50,23 +51,24 @@ TimerConfig* get_timer_config(int channel)
 
 void set_default_timer_config(TimerConfig tim_cfg[], const size_t cnt)
 {
-        for (size_t i = 0; i < cnt; ++i) {
-                TimerConfig *tc = tim_cfg + i;
-                set_default_channel_config(&tc->cfg);
-                tc->filterAlpha = TIMER_DEFAULT_FILTER_ALPHA;
-                tc->mode = MODE_LOGGING_TIMER_RPM;
-                tc->pulsePerRevolution = TIMER_DEFAULT_PULSE_PER_REV;
-                tc->timerSpeed = TIMER_MEDIUM;
-                tc->filter_period_us = TIMER_DEFAULT_FILTER;
-                tc->edge = TIMER_DEFAULT_EDGE;
-        }
+	for (int i = 0; i < cnt; ++i) {
```

Comment:
Migration to an int type was intentional because of the use of the `snprintf` command.  This way I avoid a cast.  Perhaps you want me to use `unsigned int` instead.  That is really the only reason you want me to use `size_t` right?